README.md

```diff
@@ -1,3 +1,27 @@
 # Northcoders News API
 
-If you want to re-create the project you need to create two files .env.test env.development and set an evironment variable PGDATABASE=nc_news or PGDATABASE=nc_news_test in both with the database names referred to in setup.sql
\ No newline at end of file
+### Hosted Version
+https://nc-news-a8e3.onrender.com/api
+
+### Project Outline
+This project is a news outlet where you can create delete and update a plethora of things including but not limited to comments, articles and topics.
+
+### Setup Instructions
+To clone you will need to go onto github and click the clone link which is come up and then paste that link into VS Code and hit enter. Once that is done you will need to install theses dependencies
+ - pg
+ - pg-format
+ - dotenv
+ - express
+Also you will need to install these devDependencies
+ - jest
+ - jest-sorted
+ - supertest
+ - jest extended
+
```

Comment:
These are all listed in the package.json - running `npm i` / `npm install` will install them all at once

Remember that npm install with no package named will install all the dependencies listed in the package.json!